server/cluster/metrics.go

```diff
@@ -71,13 +71,13 @@ var (
 			Help:      "Current state of the cluster",
 		}, []string{"state"})
 
-	regionWaitingListGauge = prometheus.NewGauge(
+	regionListGauge = prometheus.NewGaugeVec(
```

Comment:
Need to update `grafana.json`?

Reply:
no need. it do not use in grafana ,only for debug